packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -127,23 +137,85 @@ Future<void> checkGradleDependencies() async {
   progress.stop();
 }
 
+/// Tries to create [settings_aar.gradle] in an app project by removing the subprojects
+/// from the existing [settings.gradle] file. This operation will fail if the existing
+/// [settings.gradle] file has local edits.
+void _createSettingsAarGradle(Directory androidDirectory) {
```

Comment:
According to https://codecov.io/gh/flutter/flutter/pull/35217/diff#D21-143, this method has no test coverage